.github/workflows/release.yml

```diff
@@ -103,6 +104,24 @@ jobs:
           fi
           git tag -l -n --format="%(contents:subject)" $(git describe --tags --abbrev=0) > TAG-COMMENT.txt
 
+      - name: MSI
+        shell: bash
+        if: matrix.os == 'windows-latest'
+        run: |        
+          # Return a qldb.exe
+          cargo fetch && cargo build --release            
+          strip "target/${{ matrix.target }}/release/${{ matrix.bin }}"
+          
+          # Install cargo-wix to compile main.wxs
+          cargo install cargo-wix
+                      
+          # Create MSI based on target/wix/main.wxs
+          cargo wix 
+          
+          cd target/wix
+          # rename msi file to "qldb.msi"
+          mv amazon_qldb_shell-0.1.0-x86_64.msi ../../qldb.msi
```

Comment:
Hmmm - is the version (0.1.0) dynamic? I would assume that it will change based on...something? If so, this will break on upgrade when the version is changed.

Reply:
Also - there was an ask earlier to rename the file to qldb_shell?

Reply:
Seems like this version is from the [cargo.toml version](https://github.com/awslabs/amazon-qldb-shell/pull/61/files#diff-2e9d962a08321605940b5a657135052fbcef87b5e360662bb527c96d9a615542L3)

Reply:
Yes, it seems like 0.1.0 is from the cargo.toml.  The other os releases are named qldb-v2.0.17-mac.tar.gz for example.  Should we not named the Windows os a similar one like qldb-v2.0.17-win.msi ?

Reply:
We should have the name here be dynamic and pick up the version so it doesn't break.

Reply:
That mv command is incorrect,

it is `mv (file you want to change) (new file name)`

Currently it still renames to qldb.msi

Reply:
Changed the naming so it doesn't break.